### PR TITLE
sg hacking hour offsite: removing cloud v1 code

### PR DESCRIPTION
**Pull request by mrnugget**
Code removed in `sg hacking hour`

## Test plan

- 🤞 
- Existing tests

### CODE REVIEW

File: cmd/frontend/graphqlbackend/external_service.go [hunk 1]
```diff
@@ -192,17 +178,7 @@ func (r *externalServiceResolver) NextSyncAt() *gqlutil.DateTime {
 var scopeCache = rcache.New("extsvc_token_scope")
 
 func (r *externalServiceResolver) GrantedScopes(ctx context.Context) (*[]string, error) {
```
Comment (eseliger) [inline cmd/frontend/graphqlbackend/external_service.go:180]: remove

File: internal/repos/syncer.go [hunk 2]
```diff
@@ -877,10 +877,6 @@ func (s *Syncer) observeSync(
 		var owner string
 		if svc == nil {
 			owner = ownerUndefined
-		} else if svc.NamespaceUserID > 0 {
-			owner = ownerUser
```
Comment (eseliger) [inline internal/repos/syncer.go:881]: can you also remove these two constants?

**Review by eseliger — APPROVED** [review]

**Review by ryanslade — APPROVED** [review]

**Review by marekweb — APPROVED** [review]

**Comment by sg-e2e-regression-test-bob** [comment]
## Bundle size report 📦

| Initial size | Total size | Async size | Modules |
| --- | --- | --- | --- |
| 0.00% (0.00 kb) | -0.00% (-0.42 kb) | -0.00% (-0.42 kb) | 0.00% (0) |

Look at the [Statoscope report](https://console.cloud.google.com/storage/browser/_details/sourcegraph_reports/statoscope-reports/sg-hacking-hour-crew/remove-cloud-v1-code/compare-report.html;tab=live_object) for a full comparison between the commits e74124f and cdf3710 or [learn more](https://docs.sourcegraph.com/dev/how-to/testing#bundlesize).


<details>
<summary>Open explanation</summary>

- `Initial size` is the size of the initial bundle (the one that is loaded when you open the page)
- `Total size` is the size of the initial bundle + all the async loaded chunks
- `Async size` is the size of all the async loaded chunks
- `Modules` is the number of modules in the initial bundle
</details>

**Comment by jhchabran** [comment]
@eseliger @mrnugget I had to drop a few more things to fix the errors in CI, could you please take a careful look?

**Comment by mrnugget** [comment]
I think there's now a huge conflict with https://github.com/sourcegraph/sourcegraph/pull/44992 ... I'll let @ryanslade merge his first and then we can see if there's still stuff left.

**Comment by ryanslade** [comment]
Ok, I've merged the other PR. Not sure whether you want to try and resolve all the conflicts or just start fresh?

**Comment by mrnugget** [comment]
Okay, I rebased this PR after @ryanslade merged his. It's now just a few tiny cleanups that were missed and a reworking of one of the tests we bumped into.

**Pull request merged**